<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
  background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 1, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(252, 194, 106);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_sum_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(252, 194, 106);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Общая сумму кредита:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment_result">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(68, 68, 68);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_sum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(252, 194, 106);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(252, 194, 106);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Переплата по кредиту:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_pay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(252, 194, 106);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Первый платже</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deadline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(252, 194, 106);
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Срок:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_pay_result">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(68, 68, 68);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_sum_result">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(68, 68, 68);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ann">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
  background-color: rgb(255, 1, 57);
  color: white; 
  border: 1px solid gray;
}
QRadioButton::indicator {
    background-color: white ;
    width: 15px;
    height: 15px;
border-radius: 8px;
  border: 1px solid gray;
}
QRadioButton::indicator:checked {
    background-color: rgb(252, 194, 106) ;
    width: 15px;
    height: 15px;
border-radius: 8px;
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
  background-color: rgb(255, 1, 57);
  color: white; 
  border: 1px solid gray;
}
QRadioButton::indicator {
    background-color: white ;
    width: 15px;
    height: 15px;
border-radius: 8px;
  border: 1px solid gray;
}
QRadioButton::indicator:checked {
    background-color: rgb(252, 194, 106) ;
    width: 15px;
    height: 15px;
border-radius: 8px;
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QComboBox" name="year_or_month">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Год</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="percent_credit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sum_credit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="deadline_credit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_pay_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(252, 194, 106);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Последний платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_last_pay_result">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(68, 68, 68);;
  color: white; 
  border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
